src/scimlfunctions.jl

```diff
@@ -2102,18 +2094,12 @@ struct OptimizationFunction{iip, AD, F, G, H, HV, C, CJ, CH, LH, HP, CJP, CHP, L
     cons::C
     cons_j::CJ
     cons_h::CH
-    lag_h::LH
     hess_prototype::HP
     cons_jac_prototype::CJP
     cons_hess_prototype::CHP
-    lag_hess_prototype::LHP
     syms::S
     paramsyms::S2
     observed::O
-    hess_colorvec::HCV
-    cons_jac_colorvec::CJCV
-    cons_hess_colorvec::CHCV
-    lag_hess_colorvec::LHCV
```

Comment:
But why are these all removed?

Reply:
These are not used anywhere yet, I think it's confusing to have fields that are not used yet. We can add them back incrementally as support for these get added?

Reply:
Why not just finish using them? AutoForwardDiff is very close. It's almost quicker to do that 😅

Reply:
https://github.com/SciML/Optimization.jl/pull/374 
https://github.com/SciML/Optimization.jl/pull/366

Yeah it's close, but they are stalled so I didn't want to hold this back with the inflow of docs issues having become much more recently

Reply:
yeah but then there's no reason to delete these if they will be added right back for those PRs?

Reply:
Okay, added them back 👍